pkg/summary/summary_test.go

```diff
@@ -74,8 +57,8 @@ func TestRunSummary(t *testing.T) {
 			Namespace: "testing",
 		},
 		Spec: v1beta2.VerticalPodAutoscalerSpec{
-			TargetRef: &autoscaling.CrossVersionObjectReference{
-				APIVersion: "apps/v1",
+			TargetRef: &autoscalingv1.CrossVersionObjectReference{
+				APIVersion: "extensions/v1beta1",
```

Comment:
Why did you change the apiVersion of the deployment in this test? This apiVersion will be completely removed in kube 1.16

Reply:
I'm not exactly sure. I've changed this to "apps/v1".